PredictMelanomaServer.py

```diff
@@ -24,10 +24,26 @@
 """
 
 from flask import Flask, request, jsonify
+from flask_cors import CORS
 app = Flask(__name__)
+CORS(app)
 req_num = 0
 
 
+def send_error(message, code):
+    err = {
+        "error": message
+    }
+    return jsonify(err), code
+
+
+def validate(img):
+    if img.shape[2] == 3:
```

Comment:
A docstring for this would be important, to convey why you expect this array shape to be 3 all the time.

Reply:
Do you need this to raise any exceptions when `False`?

Reply:
My thought is get_prediction requires an array of (x, y, 3) so if it doesn't have 3 I could raise an error beforehand?

Reply:
Sounds good... just pointing out that someone might not easily recognize why this would be the case (e.g., couldn't use a CMYK image).